app/single_resource/views.py

```diff
@@ -150,15 +151,32 @@ def edit(resource_id):
                   'form-error')
     # Field id is not needed for the form, hence omitted with [1:].
     for field_name in resource_field_names[1:]:
-        form[field_name].data = resource.__dict__[field_name]
+        if form[field_name]:
```

Comment:
was getting an error that the field_name wasn't there for some of the resources? so i thought it was safe to do an if check